dist/rpm/os-autoinst.spec

```
@@ -68,7 +68,7 @@ Source0:        %{name}-%{version}.tar.xz
 %bcond_with black
 %endif
 # SLE is missing Python support requirements
-%if !0%{?is_opensuse}
+%if 0%{?is_opensuse}
```

Ah. I guess you need to get rid of the ? here 🤔 
```suggestion
%if 0%{is_opensuse}
```

---

Nope `0%{?is_opensuse}` is correct syntax: https://en.opensuse.org/openSUSE:Specfile_guidelines#Conditionals